### PR TITLE
Feature lua float function

### PR DESCRIPTION
**Pull request by nordinvannes**

### CODE REVIEW

File: scripter/lua/lua.go [hunk 1]
```diff
@@ -126,6 +126,17 @@ func (l *luaScripter) SetStringFunction(name string, getString func() string) er
 	return nil
 }
```
Comment (Icety) [inline scripter/lua/lua.go:177]: Comments maybe?

**Comment by Icety** [comment]
Looks good

**Review by Icety — APPROVED** [review]
Looks good, only no GoDoc added to the SetFloatFunction

File: scripter/lua/lua.go (hunk 1 above)
Comment (nordinvannes) [reply scripter/lua/lua.go:177]: Done.

**Pull request merged**